Sources/Sentry/SentryTracer.m

```diff
@@ -540,15 +583,15 @@ - (SentryTransaction *)toTransaction
         [_children addObjectsFromArray:appStartSpans];
         spans = [_children copy];
     }
+    transaction.spans = spans;
+
+    transaction.timestamp = self.timestamp;
```

Comment:
`m`: Can't we move that to `createTransaction`?

Reply:
no, because the child spans haven't been created by that point, and `timestamp` I believe is the point where everything is finished, so we can't set that at the start. This is the same point in execution where these were always being set.